google-cloud-pubsub/lib/google/cloud/pubsub/subscriber.rb

```diff
@@ -55,8 +91,15 @@ def initialize subscription_name, callback, deadline: nil, streams: nil,
           super() # to init MonitorMixin
         end
 
+        ##
+        # Starts the subscriber.
```

Comment:
I don't know if the method is the right place for that, or if some guide/overview should explain that. I don't have a great vision for what users want to know when looking at the method, so I kept it as brief as would be expected.

I followed Speech's [Stream#start](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-speech/v0.25.0/google/cloud/speech/stream?method=start-instance) documentation here as well.